CHANGELOG.rst

```diff
@@ -4,6 +4,15 @@ Changelog
 dev-version
 -----------
 
+v3.2.1 [13 Sep 2022]
+----------------------
+
+Changed
+~~~~~
```

Comment:
this needs to be the full length of the title. Also, you could update the deploy workflow to use the latest version of the uploader action, rather than `master` (see the error for the deploy script)